thirdparty/scripts/setup.sh

```diff
@@ -35,6 +35,11 @@ bash "$TP_SCRIPT_DIR/build_credis.sh"
 ##############################################
 bash "$TP_SCRIPT_DIR/build_boost.sh"
 
+##############################################
+# gtest
+##############################################
+bash "$TP_SCRIPT_DIR/build_gtest.sh"
```

Comment:
I think we're already building googletest 

https://github.com/ray-project/ray/blob/f69cbd35d4e86f2a3c2ace875aaf8166edb69f5d/cmake/Modules/ThirdpartyToolchain.cmake#L31-L35